model/src/main/kotlin/config/ArtifactoryCacheConfiguration.kt

```diff
@@ -20,10 +20,22 @@
 package com.here.ort.model.config
 
 import com.fasterxml.jackson.annotation.JsonProperty
+import config.CacheConfiguration
 
 data class ArtifactoryCacheConfiguration(
         val url: String,
 
         @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
         val apiToken: String = ""
-)
+
+) : CacheConfiguration {
+    override fun validate() {
+        require(url.isNotBlank()) {
+            "URL for Artifactory cache is missing."
+        }
+
+        require(apiToken.isNotBlank()) {
+            "API token for Artifactory cache is missing."
```

Comment:
I'm wondering whether this could actually be valid in case an (internal) Artifactory repository is open for writing to anyone?

Reply:
@sschuberth I'm not too familiar Artifactory tbh